weldx/transformations.py

```diff
@@ -2454,9 +2454,13 @@ def plot(self):
         plt.figure()
         color_map = []
         pos = self._get_tree_positions_for_plot()
-        nx.draw(
-            self._graph, pos, with_labels=True, font_weight="bold", node_color=color_map
-        )
+
+        graph = deepcopy(self._graph)  # TODO: Check if deepcopy is necessary
+        # only plot inverted directional arrows
+        remove_edges = [edge for edge in graph.edges if graph.edges[edge]["defined"]]
```

Comment:
This confused me. I was wondering if there was a `not` missing until I realized it is correct 😄